lropt/robust_problem.py

```diff
@@ -244,33 +224,6 @@ def _udata_to_lst(self, data):
         return u_params_mat
 
     def lagrange(self, vars, y_params_mat, u_params_mat, alpha, lam, eta=0.05, kappa=-0.05):
```

Comment:
This function should:
1. be called `lagrangian`
2. return only F + lam @ H

the other two parts should be called `evaluation_loss` and `prob_constr_violation` and executed separately from this function